utils/check_quickstart_uniqueness.ts

```diff
@@ -1,26 +1,28 @@
-import type { QuickstartConfig } from './types/QuickstartConfig';
-import type { FilePathAndContents } from './helpers';
-
-import {
+'use strict';
+const {
   readQuickstartFile,
   removeRepoPathPrefix,
   findMainQuickstartConfigFiles,
-} from './helpers';
+} = require('./helpers');
```

Comment:
I think it might be more consistent if we use imports here instead of require.

Reply:
Strange i dont remember doing this, and not sure why i did.  Let me go switch it back